### PR TITLE
Changelog v1.17.0

### PR DESCRIPTION
**Pull request by 6543**

### CODE REVIEW

**Review by KN4CK3R — APPROVED** [review]

**Review by noerw — APPROVED** [review]

**Review by jolheiser — CHANGES_REQUESTED** [review]
Just blocking until ready. 🙂

**Comment by lunny** [comment]
Could we wait for #20548 and #20549 ? Some users need that for v1.17.

**Comment by Gusted** [comment]
> Could we wait for #20548 and #20549 ? Some users need that for v1.17.

Backporting features on the last moment? Looks risky for me, no time to iron out bugs etc.

**Comment by 6543** [comment]
we did reject a lot of "can we have this in v1.17" ... at that rate we just could v1.18 ... - I'm also against it

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,44 +4,7 @@ This changelog goes through all the changes that have been made in each release
...
-  * Check if project has the same repository id with issue when assign project to issue (#20133) (#20188)
-  * Fix remove file on initial comment (#20127) (#20128)
-  * Catch the error before the response is processed by goth (#20000) (#20102)
-  * Dashboard feed respect setting.UI.FeedPagingNum again (#20094) (#20099)
-  * Alter hook_task TEXT fields to LONGTEXT (#20038) (#20041)
-  * Respond with a 401 on git push when password isn't changed yet (#20026) (#20027)
-  * Return 404 when tag is broken (#20017) (#20024)
-
-## [1.17.0-rc1](https://github.com/go-gitea/gitea/releases/tag/v1.17.0-rc1) - 2022-06-18
+## [1.17.0](https://github.com/go-gitea/gitea/releases/tag/v1.17.0) - 2022-07-29
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.17.0](https://github.com/go-gitea/gitea/releases/tag/v1.17.0) - 2022-07-30
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -222,13 +188,43 @@ been added to each release, please refer to the [blog](https://blog.gitea.io).
...
   * Prettify number of issues (#17760)
   * Add a "admin user generate-access-token" subcommand (#17722)
-  * Move project files into models/project sub package (#17704)
   * Custom regexp external issues (#17624)
   * Add smtp password to install page (#17564)
   * Add config options to hide issue events (#17414)
   * Prevent double click new issue/pull/comment button (#16157)
   * Show issue assignee on project board (#15232)
 * BUGFIXES
+  * Ensure that all unmerged files are merged when conflict checking (#20528) (#20536)
```
Comment (6543) [inline CHANGELOG.md:198]:
```suggestion
  * WebAuthn CredentialID field needs to be increased in size (#20530) (#20555)
  * Ensure that all unmerged files are merged when conflict checking (#20528) (#20536)
```

**Review by jolheiser — APPROVED** [review]

**Review by noerw — APPROVED** [review]

**Pull request merged**